src/com/gfredericks/test/chuck/properties/macros.clj

```diff
@@ -0,0 +1,18 @@
+(ns com.gfredericks.test.chuck.properties.macros
+  "Alternative to clojure.test.check.properties."
+  (:require [com.gfredericks.test.chuck.properties.impl :refer [for-bindings]]
+            [com.gfredericks.test.chuck.generators :as gen']))
+
+(defmacro for-all
+  "Alternative version of clojure.test.check.properties/for-all where
+  the binding forms are interpreted as per
+  com.gfredericks.test.chuck.generators/for."
+  [bindings expr]
+  (let [bound-names (for-bindings bindings)
+        quoted-names (map #(list 'quote %) bound-names)]
+    `(~'cljs.test.check.properties/for-all [{:syms [~@bound-names]}
+                    (gen'/for ~bindings
+                      (with-meta
+                        ~(zipmap quoted-names bound-names)
+                        {::for-all-bindings-map true}))]
```

Comment:
Yes, it's not being used anywhere in the project. When it was [introduced](https://github.com/gfredericks/test.chuck/commit/066d7bffc1a38c8842e5560d28c822dedea88b9c) it was supposed to be used by tooling. We could try asking the guy who did that commit :P, or if he doesn't remember if he ended using it, we could just remove it and see if someone complains. I couldn't find a reference to it [anywhere in github](https://github.com/search?q=for-all-bindings-map&type=Code&utf8=%E2%9C%93)

But yes, I think it's better to remove it in a separate commit, because we would remove it from clj (already released) and cljs side, right?